exercises/sheet4/marta_bustins_marta_gonzalez_esteve_tarrago/4.tex

```diff
@@ -0,0 +1,186 @@
+    &=\lambda x+\left(1-\lambda\right)x\qquad\qquad\qquad\qquad\quad\;\mathrm{Since\,}\gamma\mathrm{\,and\,}\rho\mathrm{\,are\,sections}\\
+    &=x\\
+\end{align*}
+so any convex combination of sections is a section (seen just by 2 sections, but repeating this argument by induction can be easily proved). In particular, this shows that $\Sigma\left(P,Q\right)$ is convex, since for any $x$, $y\in\Sigma\left(P,Q\right)$, $x=\frac{1}{\mathrm{volQ}}\int_Q\gamma\left(s\right)ds$, $y=\frac{1}{\mathrm{volQ}}\int_Q\rho\left(s\right)ds$, $\gamma$, $\rho$ two sections and $\lambda\in\left[0,1\right]$ we get that
+    $$\lambda\frac{1}{\mathrm{volQ}}\int_Q\gamma\left(s\right)ds+\left(1-\lambda\right)\frac{1}{\mathrm{volQ}}\int_Q\rho\left(s\right)ds=\frac{1}{\mathrm{volQ}}\left(\int_Q\left(\lambda\gamma\left(s\right)\left(1-\lambda\right)\rho\left(s\right)\right)ds\right)\in\Sigma\left(P,Q\right)$$
+since convex combinations of sections is a section, as we have seen. Moreover, $\Sigma\left(P,Q\right)\subseteq\pi^{-1}\left(r_0\right)$ since, for any $x\in\Sigma\left(P,Q\right)$, $x=\frac{1}{\mathrm{volQ}}\int_Q\gamma\left(s\right)ds$, $$\pi\left(x\right)=\pi\left(\frac{1}{\mathrm{volQ}}\int_Q\gamma\left(y\right)dy\right)=\frac{1}{volQ}\int_Q\pi\left(\gamma\left(y\right)\right)dy=\frac{1}{volQ}\int_Qydy=r_0.$$ On the other hand, $\pi\colon\mathbb{R}^p\to\mathbb{R}^q$ as afine function can be written as $\pi=\pi_l+r_0$, where $\pi_l\colon\mathbb{R}^p\to \mathbb{R}^q$ is a linar function. Hence, in order to find $\mathrm{dim}\pi^{-1}\left(r_0\right)$ it suffices to find $\mathrm{dim}\pi^{-1}_l\left(0\right)=\mathrm{dim\,ker}\pi_l$. By the first isomorphism theorem we have $$\mathrm{dim\,ker}\pi_l=\mathrm{dim\,P}-\mathrm{dim\,Q}.$$ Thus, $\Sigma\left(P,Q\right)$ has dimension $\mathrm{dim\,P}-\mathrm{dim\,Q}$ at most.
+
+Since tight sections correspond to some subsets of $\mathcal{L}\left(P\right)$, there will be a finite number of tight sections.
+
+\textit{Claim.} Every section $\gamma$ not tight can be expressed as a convex combination of 2 other sections.
+
+As we have seen, $\Sigma\left(P,Q\right)$ is convex, whose points are given by the sections $\gamma\colon Q\to P$. Then, since every interior point of a convex set can be expressed as convex combinations of other points of the set, but the vertices, and the only sections that cannot be written as convex combinations of other sections are the tight ones, we have that the vertices of $\Sigma\left(P,Q\right)$ correspond to the tight sections. Then, $\Sigma\left(P,Q\right)$ is the convex hull of a finite number of points, so it must be a polytope.
+
+%\textcolor{red}{Feu-li una ullada! Fins aquí és tot el que tenia al paper jo, ara ja toca veure que, de fet, les tight sections corresponen a les $\pi$-coherent tight subdivisions... Si hi ha algo que no s'entén canvieu-ho :) }
+\end{document}
```

Comment:
Please separate your sentences and relevant formulae into their own lines to improve readability!

Thanks, and thank you for your hard work!
Best,
Julian